client/main.go

```diff
@@ -129,27 +149,28 @@ func run(addr, id string) error {
 		pixel.R(0, 64, 64, 128),
 		pixel.R(64, 64, 128, 128),
 	}
-
 	playerSprite := pixel.NewSprite(playerSheet, playerFrames[2])
 
-	animationRate := 10.0 // framerate of player animation
-	elapsed := 0.0
-	fps := 0
+	animationRate := 1.0 // framerate of player animation
```

Comment:
slow down animation to 1 fps, seems to help with jitter for now (until changing player sprite)